src/lib/locals.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  getGTM: function (req, res, next) {
+    res.locals.gtmId = req.app.get("APP.GTM.ID");
+    res.locals.scriptNonce = req.app.get("APP.GTM.SCRIPT_NONCE");
```

Comment:
More in relation to helmet but this inline script
document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
tries to read nonce from a field called cspNonce

Reply:
That's a good point. We're not using that part of the GOV.UK frontend template, but it'd be better to keep using their naming standards. I've updated it accordingly.